src/plugins/vis_type_timeseries/public/application/components/vis_editor.tsx

```diff
@@ -163,8 +168,16 @@ export class VisEditor extends Component<TimeseriesEditorProps, TimeseriesEditor
     this.visDataSubject.next(visData);
   };
 
+  dismissNotice = () => {
+    this.localStorage.set('CALLOUT_HIDDEN', true);
```

Comment:
nit: i think name should be more specific cause it stored in local storage with that name
![image](https://user-images.githubusercontent.com/20072247/123960687-4339bf80-d9b8-11eb-811b-df101cd64492.png)